barretenberg/cpp/Earthfile

```diff
@@ -29,6 +29,11 @@ preset-release:
     RUN cmake --preset clang16 -Bbuild && cmake --build build --target bb && rm -rf build/{deps,lib,src}
     SAVE ARTIFACT build/bin
 
+preset-release-pic:
+  FROM +source
+  RUN cmake --preset clang16-pic -Bbuild && cmake --build build --target world_state_napi && cp ./build/lib/world_state_napi.node ./build/bin && rm -rf build/{deps,lib,src}
```

Comment:
not a fan of the pic name, this is a low level detail for a high level feature. This preset and this earthly command could likely be preset-aztec-world-state or similar

Reply:
Renamed